<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园导游咨询模拟系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="MyQGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="menu_Widget">
      <property name="minimumSize">
       <size>
        <width>204</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>204</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="developer">
       <attribute name="title">
        <string>开发者功能</string>
       </attribute>
       <widget class="QPushButton" name="add_edge_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>add_edge</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_pos_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>add_pos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_poss_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>show_poss</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clear_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_edges_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>show_edges</string>
        </property>
       </widget>
       <widget class="QPushButton" name="show_places_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>310</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>show_places</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="key_Edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="key_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>请输入密码：</string>
        </property>
       </widget>
       <widget class="QLabel" name="key_label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;请输入密码以用开发者功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="add_palce">
       <attribute name="title">
        <string>修改地点</string>
       </attribute>
       <widget class="QPushButton" name="add_place_btu">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>添加地点</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>地点名：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>361</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;使用说明：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;添加地点：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;在右侧点击待添加地点&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;后，并再输入框中输入&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;地点名称即可添加。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;删除地点：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;在输入框中输入待删除&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;地点，点击删除地点即&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;可删除。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;注意：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;不可以添加相同地点&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;名称！！！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="name_Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="erase_place_btu">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除地点</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="query">
       <attribute name="title">
        <string>查询</string>
       </attribute>
       <widget class="QPushButton" name="query_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>查询路线</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>起点：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>终点：</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="short_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>最短距离路线</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="far_btu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>最高卡路里路线</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QComboBox" name="snameBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="enameBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>171</width>
          <height>161</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;使用说明：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;输入起点、终点以及查&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;询状态后，点击查询路&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;线按钮可&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 路程：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>卡路里消耗：</string>
        </property>
       </widget>
       <widget class="QLabel" name="len_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="k_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>270</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>myqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
